eng/pipelines/official/stages/publish.yml

```diff
@@ -28,7 +27,6 @@ stages:
 # Stages-based publishing entry point
 - template: /eng/common/templates-official/post-build/post-build.yml
   parameters:
-    publishingInfraVersion: ${{ parameters.publishingInfraVersion }}
```

Comment:
unrelated change?

Reply:
Yes. The publishing infra version doesn't need to be set anymore.